docs/core/diagnostics/FAQ-dumps.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
This new file will need an entry in the table of contents.

Reply:
Also, you might want to convert this to a YAML file to optimize the questions and answers for a search engine: https://review.learn.microsoft.com/en-us/help/contribute/contribute-how-to-faq-guide?branch=main.

Reply:
Finally, all Markdown files need to be named using lower-case characters only.